activesupport/lib/active_support/duration.rb

```diff
@@ -180,13 +180,15 @@ def since(time = ::Time.current)
       sum(1, time)
     end
     alias :from_now :since
+    alias :after :since
 
     # Calculates a new Time or Date that is as far in the past
     # as this Duration represents.
     def ago(time = ::Time.current)
       sum(-1, time)
     end
     alias :until :ago
+    alias :before :ago
```

Comment:
I'm happy to make this change if it's desired, but as you say, perhaps in a different PR. What is the process for deprecating something like that?